fits/FITS/blockio.cc

```diff
@@ -73,7 +74,14 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
 	strcpy(m_filename,f);
 	// using cfitsio of NASA to open fits file for writting and reading.
 	int l_status, iomode;
-	l_status = 0; 
+	File myfile(m_filename);
+	if (myfile.exists() && myfile.isReadable()) {
```

Comment:
`isReadable` also checks existance, so the existance check is unnecessary (but also harmless)